measures/comstock_sensitivity_reports/measure.rb

```diff
@@ -179,6 +184,9 @@ def energyPlusOutputRequests(runner, user_arguments)
     result << OpenStudio::IdfObject.load("Output:Variable,*,Water Heater Propane Energy,RunPeriod;").get # J
     result << OpenStudio::IdfObject.load('Output:Variable,*,Water Heater Heating Energy,RunPeriod;').get # J
     result << OpenStudio::IdfObject.load('Output:Variable,*,Water Heater Unmet Demand Heat Transfer Energy,RunPeriod;').get # J
+    result << OpenStudio::IdfObject.load("Output:Variable,*,Unitary System DX Coil Cycling Ratio,#{timeseries_timestep};").get # -
+    result << OpenStudio::IdfObject.load("Output:Variable,*,Unitary System Sensible Cooling Rate,#{timeseries_timestep};").get # W
```

Comment:
I am not as familiar with these unitary system variables. If we are just using it to determine if there is heating or cooling, then maybe it's fine. But if we are using it for the calculation, I would prefer using the heating and cooling rate from the coil objects.

Reply:
@ChristopherCaradonna that may cause: building with 30 pszs ending up with 60 (30 for cooling and 30 for heating) timeseries (each with 8760 datapoints) for that specific model, maybe? burden on simulation workflow? I'll check what the document says for the variables above. Let me know if my example with 60 timeseries output vars seems really what's gonna happen.